shared/app/nav.native.js

```diff
@@ -352,7 +352,17 @@ class Nav extends Component<Props, {keyboardShowing: boolean}> {
       />
     )
     const layerScreens = this.props.routeStack.filter(r => r.tags && r.tags.layerOnTop)
-    const layers = layerScreens.map(r => r.leafComponent({shouldRender: true}))
+    const layers = layerScreens.map(
```

Comment:
these totally ignored the tags! Not adding more right now but likely this needs to change

Reply:
These mess up the send form on iPhone X, the back button on the main form and 'Cancel' on the qr scan are unreachable

Reply:
iphone x stuff should be fixed in master and merged back into here